client/pages/search.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+
+export default class Search extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchTerm: '',
+      plantData: [],
+      hasUserTyped: false
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  componentDidMount() {
+    fetch('http://harvesthelper.herokuapp.com/api/v1/plants/?api_key=349d9448a25ad42ce213e1018c428cd3')
```

Comment:
Instead of hard-coding your API key, you should put it in your `.env` file and then dynamically replace it with the Webpack Environment plugin. Instead of a string, you'd do `apiKey: process.env.HARVEST_HELPER_API_KEY` here in your JavaScript.

Here is the Environment plugin.
https://webpack.js.org/plugins/environment-plugin/#usage

Here is how you register a plugin in your Webpack config file.
https://webpack.js.org/concepts/plugins/#usage

Lemme know if I can help 👍